pipelines/utilities/run_longhorn_test.sh

```diff
@@ -117,6 +120,9 @@ run_longhorn_upgrade_test(){
     yq e -i 'select(.spec.containers[0] != null).spec.containers[0].env[6].value="true"' ${LONGHORN_UPGRADE_TESTS_MANIFEST_FILE_PATH}
   fi
 
+  ## for v2 volume test
+  yq e -i 'select(.spec.containers[0].env != null).spec.containers[0].env += {"name": "RUN_V2_TEST", "value": "'${RUN_V2_TEST}'"}' "${LONGHORN_UPGRADE_TESTS_MANIFEST_FILE_PATH}"
+
```

Comment:
Do you know why sometimes v2 related code flow doesn't need to rely on this `RUN_V2_TEST` value? Like https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L3784

Reply:
The code flow to decide which data engine is using for test moved to: https://github.com/longhorn/longhorn-tests/blob/82d675991f3c211eb94328d285b75b4b1301dc6c/manager/integration/tests/common.py#L335-L339

Reply:
Could you confirm the `SETTING_V2_DATA_ENGINE` setting in https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L3784 is controlled by the code flow you mentioned above?

Reply:
Hi @yangchiu, could you please see my reply [here](https://github.com/longhorn/longhorn-tests/pull/2405#discussion_r2022140771)? Thank you!